crud/compare/plan.lua

```diff
@@ -286,6 +286,131 @@ function plan.new(space, conditions, opts)
     return plan
 end
 
+function plan.new_readview(space, conditions, opts)
+    dev_checks('table', '?table', {
+        first = '?number',
+        after_tuple = '?table|cdata',
+        field_names = '?table',
+        force_map_call = '?boolean',
+        sharding_key_as_index_obj = '?table',
+        bucket_id = '?number|cdata',
+    })
+
+    conditions = conditions ~= nil and conditions or {}
+    opts = opts or {}
+
+    local space_name = space.name
+    local space_indexes = space.index
+    local space_format = space.format
```

Comment:
It's not worth to create a whole new `plan` method if the only difference is format extraction.

Reply:
fixed